src/app/forms/custom-sections/letters/letters.component.ts

```diff
@@ -85,6 +85,7 @@ export class LettersComponent implements OnInit, OnDestroy, OnChanges {
   get correctApprovalType(): boolean {
     return (
       this.techRecord?.techRecord_approvalType !== undefined &&
+      this.techRecord?.techRecord_approvalType !== null &&
       (Object.values(LettersIntoAuthApprovalType) as string[]).includes(this.techRecord.techRecord_approvalType!.valueOf())
```

Comment:
```suggestion
      (Object.values(LettersIntoAuthApprovalType) as string[]).includes(this.techRecord.techRecord_approvalType.valueOf())
```
I'm not sure the `!` is needed if you're checking for `null` and `undefined`. Unless the compiler has some other issue with it.